.env.example

```
@@ -17,6 +17,10 @@ UPSTASH_REDIS_REST_TOKEN=your-upstash-token
 # Auth
 # Sinh random: openssl rand -base64 32
 JWT_SECRET=change-me-to-a-random-string
+RESEND_API_KEY=re_xxxxxxxxxxxxxxxxxxxxxxxxx
+EMAIL_FROM=example@gmail.com
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The EMAIL_FROM key should go before the JWT_SECRET key

(UnorderedKey)